src/cap.rs

```diff
@@ -9,7 +9,11 @@ use kvm_bindings::*;
 
 /// Enumeration of the extension capability list that KVM exposes.
 ///
-/// For details check the
+/// The capabilities list can be used in conjunction with
+/// [Kvm::check_extension()](struct.Kvm.html#method.check_extension) to check if a particular
+/// capability is available.
+///
+/// The list of capabilities is based on the the KVM_CAP_* defines from the
 /// [Linux KVM header](https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/kvm.h).
```

Comment:
Could we do the same thing (put a link to the elixir sources) for the `VpuExit` documentation. Currently it looks like this : `The exit reasons are mapped to the KVM_EXIT_* defines from include/uapi/linux/kvm.h.` and it looks kind of ugly when doing `cargo doc --open`.